uwblueprint/sistering
# Remove user panel mutation + fix panel UI overflow

## Ticket link

Closes #415 
Closes #574 


## Implementation description
* Modified prisma schema to cascade delete a user's signups.
* Hooked up the "Remove user" button with the remove volunteer/admin mutation depending on the user's role
* IMPORTANT: Do NOT delete a seed user! The remove user mutation also removes them from Firebase.
* Refetched table when a user is removed.
* Fixed UI overflow on side panel when many tags are present.



## Steps to test
1. Run `npx prisma migrate dev` for the latest table changes
2. Create a new admin or volunteer account (Do not use seed accounts!!)
3. Remove the user using a super admin account
4. Check that the user is removed from SQL tables and Firebase



## What should reviewers focus on?
* Correct prisma schema updates


## Checklist
